docs/nessie.md

```diff
@@ -47,7 +47,7 @@ From Spark 3.3 (with scala 2.12), Nessie SQL extensions can be used to manage th
 
 ```
 bin/spark-sql 
-  --packages "org.apache.iceberg:iceberg-spark-runtime-3.3_2.12:{{% icebergVersion %}},org.projectnessie:nessie-spark-extensions:{{% nessieVersion %}}"
+  --packages "org.apache.iceberg:iceberg-spark-runtime-3.3_2.12:{{% icebergVersion %}},org.projectnessie.nessie-integrations:nessie-spark-extensions-3.3_2.12:{{% nessieVersion %}}"
```

Comment:
I believe the release owner will bump the `nessieVersion` in the iceberg-docs repo https://github.com/apache/iceberg-docs/blob/457099ffe612977ad272ab142a702018bf3bfcf9/docs/config.toml#L12

Reply:
Maybe a good idea to add it to the how to release guide: https://iceberg.apache.org/how-to-release/

I'm thinking of adding the Flink version there as well.

Reply:
Ack. 

Agree that we do need this information about rising a PR for version updates here https://iceberg.apache.org/how-to-release/#documentation-release

Reply:
> Maybe a good idea to add it to the how to release guide: https://iceberg.apache.org/how-to-release/ I don't see it mentioned explicitly.

Done. Added a PR at iceberg-docs repo now.
https://github.com/apache/iceberg-docs/pull/206/files